test/clientmetricreport/DefaultClientMetricReport.test.ts

```diff
@@ -229,6 +230,37 @@ describe('DefaultClientMetricReport', () => {
     });
   });
 
+  describe('jitterBufferMs', () => {
+    const metricName = 'this-name-is-ignored';
+
+    it('returns 0 if jitter buffer delay is 0', () => {
+      const ssrc = 1;
+      const report = new StreamMetricReport();
+      report.currentMetrics['jitterBufferDelay'] = 0;
+      report.currentMetrics['jitterBufferEmittedCount'] = 1;
+      clientMetricReport.streamMetricReports[ssrc] = report;
+      expect(clientMetricReport.jitterBufferMs(metricName, ssrc)).to.equal(0);
+    });
+
+    it('returns 0 if jitter buffer emitted count is 0', () => {
+      const ssrc = 1;
+      const report = new StreamMetricReport();
+      report.currentMetrics['jitterBufferDelay'] = 1;
+      report.currentMetrics['jitterBufferEmittedCount'] = 0;
+      clientMetricReport.streamMetricReports[ssrc] = report;
+      expect(clientMetricReport.jitterBufferMs(metricName, ssrc)).to.equal(0);
+    });
+
+    it('returns the jitter buffer ms from the stream metric reports', () => {
+      const ssrc = 1;
+      const report = new StreamMetricReport();
+      report.currentMetrics['jitterBufferDelay'] = 1;
+      report.currentMetrics['jitterBufferEmittedCount'] = 2;
+      clientMetricReport.streamMetricReports[ssrc] = report;
+      expect(clientMetricReport.jitterBufferMs(metricName, ssrc)).to.equal((1 / 2) * 1000);
```

Comment:
how did we come up with this math?

Reply:
They are from audio team: https://quip-amazon.com/Gd5NAAytXv8a

Reply:
thanks